low-level/framework/sspl_reinit

```diff
@@ -60,8 +60,13 @@ chmod 440 /etc/sudoers.d/sspl
 # removed by puppet-agent (Ref: LRL-495)
 PUPPET_CONF="/etc/puppetlabs/puppet/puppet.conf"
 [ -f $PUPPET_CONF ] && {
-    PUPPET_NOOP="noop = true"
-    grep -q "$PUPPET_NOOP" $PUPPET_CONF || echo "$PUPPET_NOOP" >> $PUPPET_CONF
+    grep -q "noop\s*=.*" $PUPPET_CONF
+    if [ $? -eq 0 ]
+    then
+        sed -i -E 's/noop\ *=\ *false/noop = true/g' $PUPPET_CONF
+    else
+        echo "noop = true" >> $PUPPET_CONF
+    fi
```

Comment:
```shell
PUPPET_CONF="/etc/puppetlabs/puppet/puppet.conf"
[ -f $PUPPET_CONF ] && {
    grep -q "noop = false" $PUPPET_CONF && sed -i "s/noop = false/noop = true/" $PUPPET_CONF
    grep -q "noop = true" $PUPPET_CONF || echo "noop = true" >> $PUPPET_CONF
}
```
@mariyappanp is suggesting this simpler way considering modern shell script.
But I wanted to keep readability and consider the cases with multiple spaces before & after the '=' sign.
Everyone, please suggest which option we should go with.